packages/flutter/test/material/page_test.dart

```diff
@@ -25,17 +25,257 @@ void main() {
     await tester.pump();
     await tester.pump(const Duration(milliseconds: 1));
 
-    final Opacity widget2Opacity =
+    Opacity widget2Opacity =
```

Comment:
nit: i don't think this newline really help readability since it makes this line not match the others